README.md

```diff
@@ -24,6 +24,68 @@ Library for accessing [libkiwix](https://github.com/kiwix/libkiwix) and [libzim]
 
 AAR file will be generated in directory `lib/build/outputs/aar`
 
+### Load zim file
+
+To load a ZIM file you need to create an `Archive` object.
+
+```kotlin
+val archive = Archive("your-file-path")
+```
+
+### Load main page
+
+The `mainPage` property is used to retrieve the path of the main entry page for a Kiwix content archive.
+If the main entry is a redirect, it will fetch the path of the redirected item;
+otherwise, it will return the path of the main entry itself.
+If the main entry is not found, the archive will throw an `EntryNotFoundException`.
+
+```kotlin
+val mainPage: String?
+    get() =
+        try {
+            if (jniKiwixReader.mainEntry.isRedirect) {
+                jniKiwixReader.mainEntry.getItem(true).path
+            } else {
+                jniKiwixReader.mainEntry.path
+            }
```

Comment:
You can use `jniKiwixReader.mainEntry.getItem(true).path` all the time.
If the entry is not a redirect, `getItem` with return the item associated to the entry. And they have by definition the same path.

Reply:
Done.